pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -146,6 +175,7 @@ func (ks *scaler) handleScaleToZero(pa *pav1alpha1.PodAutoscaler, desiredScale i
 	if config.EnableScaleToZero == false {
 		return 1, true
 	}
+	key := pa.GetNamespace() + "/" + pa.GetName()
```

Comment:
I'd rather not bake in the knowledge that this takes a key.  Perhaps we can have `enqueueCB` take a `PodAutoscaler` and build the key inside?

Reply:
We don't even need to build the key, but it will require a change to the pkg.